carla-carma-integration/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
for consistency's sake, I'd recommend naming this `build-image.sh` That's the name we use in all the other repos for our docker image build scripts.

Reply:
The file name has been changed to `build-image.sh`